src/auth/pluggable-auth-client.ts

```diff
@@ -69,7 +72,9 @@ export class ExecutableError extends Error {
   readonly code: string;
 
   constructor(message: string, code: string) {
-    super(message);
+    super(
+      `The executable failed with exit code: ${code} and error message: ${message}.`
+    );
     this.name = 'ExecutableError';
     this.code = code;
     Object.setPrototypeOf(this, ExecutableError.prototype);
```

Comment:
We should mention that expiration_time is optional near the response samples on line 140ish